clang-tools-extra/test/clang-doc/invalid-options.cpp

```diff
@@ -0,0 +1,29 @@
+/// Invalid output path (%t is a file, not a directory).
+// RUN: rm -rf %t && touch %t
+// RUN: not clang-doc %s -output=%t/subdir 2>&1 | FileCheck %s
+// CHECK: clang-doc error:
+// CHECK: {{(Not a directory|no such file or directory)}}
+
+/// Invalid format option.
+// RUN: not clang-doc %s --output=%t.dir -format=badformat 2>&1 | FileCheck %s --check-prefix=BAD-FORMAT
```

Comment:
If you're going to use %t.dir, you have to `rm -rf %t.dir` at the top of the test. Alternativly, you can rewrite the preamble to make things easier

```
RUN: rm -rf %t && mkdir %t && touch %t/file
```
Then in the checks you can use %t/file if you need a dir that isn't a dir.